### DISCUSSION TITLE
Export to fat jar?

### DISCUSSION THREAD

**Discussion by cayhorstmann** [jbangdev/jbang#1833] [Ideas]
Could `jbang export` have an option `fat` in addition to `local` and `portable`?

**Comment by cayhorstmann** [comment]
Never mind--I just discovered the experimental fatjar option. Might be good to document it on https://www.jbang.dev/documentation/guide/latest/exporting.html

**Reply by maxandersen** [reply]
Wanna make a pr for the docs ? :)
